src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+    private static final String HEADER = "type,fruit,quantity";
+    private static final String BANANA_KEY = "banana";
+    private static final String APPLE_KEY = "apple";
+    private static FileWriterService fileWriterService;
+    private static String correctNameFile;
+    private static String data;
```

Comment:
I think we should make variables `correctNameFile` and `data` or constants or local variables